### PR TITLE
Disallow the entry of multiple encounter pages on the same date

### PR DESCRIPTION
**Pull request by slubwama**
https://trello.com/c/HJPyojtk/315-disallow-the-entry-of-multiple-encounter-pages-on-the-same-date

### CODE REVIEW

**Comment by slubwama** [comment]
@ssmusoke Kindly Review

**Comment by ssmusoke** [comment]
@slubwama Leverage this feature instead of writing custom code https://issues.openmrs.org/browse/HTML-262

**Comment by slubwama** [comment]
@ssmusoke Documented updated to validate the reason why this approach other than the approach suggested in the comment

**Comment by slubwama** [comment]
@ssmusoke  ETA

File: omod/src/main/webapp/resources/js/aijar.js [hunk 1]
```diff
@@ -349,3 +351,56 @@ var fieldHelper = {
...
+                                    window.location.href = url;
+                                }, 2000);
+
+                            }
+                        );
+
+                        //clear the date and continue entering the form
+                        jq(encounterDate).val('');
+                    }
+                } else {
```
Comment (ssmusoke) [inline omod/src/main/webapp/resources/js/aijar.js]: @slubwama this else is empty so is not needed

Comment (slubwama) [reply omod/src/main/webapp/resources/js/aijar.js]: @ssmusoke  was this PR Complete seems like there are comments that were not attended to??

**Pull request merged**